packages/server-wallet/src/protocols/__test__/open-channel.test.ts

```diff
@@ -36,22 +42,25 @@ const fundChannelAction1 = fundChannel({
 const fundChannelAction2 = {...fundChannelAction1, expectedHeld: BN.from(5)};
 
 test.each`
-  supported        | latestSignedByMe | latest           | funding      | action                       | cond                                                                     | result
-  ${prefundState}  | ${prefundState}  | ${prefundState}  | ${funded}    | ${signState(postfundState2)} | ${'when the prefund state is supported, and the channel is funded'}      | ${'signs the postfund state'}
-  ${prefundState}  | ${prefundState}  | ${prefundState}  | ${notFunded} | ${fundChannelAction1}        | ${'when the prefund state is supported, and alice is first to deposit'}  | ${'submits transaction'}
-  ${prefundState1} | ${prefundState1} | ${prefundState1} | ${funded}    | ${fundChannelAction2}        | ${'when the prefund state is supported, and alice is secont to deposit'} | ${'submits transaction'}
+  supported           | latestSignedByMe    | latest              | funding      | action                         | cond                                                                     | result
```

Comment:
As nice as it is to look at this table, statically, this is far too difficult to review :(

Maybe the `test.each(cases)` API is better for us. (It also means the cases can be typed, but I don't know if the test callback can take advantage of that.

Reply:
<img width="782" alt="Screen Shot 2020-10-27 at 9 41 43 AM" src="https://user-images.githubusercontent.com/8432675/97325456-a0cda900-1838-11eb-8ece-5babff6cc646.png">

You do get typed cases!

Reply:
(Let's table this discussion for another day...)